Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM quay.io/centos/centos:stream8
 
-RUN dnf install --best --refresh -y \
+RUN dnf install --best --refresh -y 
```

Comment:
can you add that removed `\` in the end?

Reply:
ohhh i see, i removed the wrong /